corehq/apps/app_manager/templates/app_manager/partials/case_list_lookup.html

```diff
@@ -108,6 +108,47 @@
                 {% endwith %}
                 <!-- /ko -->
             </div>
+            <div class="form-group">
+                <label class="col-sm-2 control-label">
+                    {% trans "Attach results to case list" %}
+                </label>
+                <div class="col-sm-4 checkbox">
+                    <input type="checkbox" data-bind="checked: lookup_attach_results"/>
+                </div>
+            </div>
+            <div data-bind="visible: lookup_attach_results">
+                <div class="form-group">
+                    <label class="col-sm-2 control-label">
+                        {% trans "Display results in case list" %}
+                    </label>
+                    <div class="col-sm-4 checkbox">
+                        <input type="checkbox" data-bind="checked: lookup_display_results"/>
+                    </div>
+                </div>
+                <div data-bind="visible: lookup_display_results">
+                    <div class="form-group" id="{{ detail.type }}-results-text">
+                        <label class="control-label col-sm-2">
+                            {% trans "Results display text" %}
+                        </label>
+                        <div class="col-sm-4">
+                            <input class="form-control" type="text"
```

Comment:
It's `input_trans` in xform_extras. The case list menu item label uses it: https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/app_manager/templates/app_manager/partials/case_list_setting.html#L24